dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/task/AbstractTask.java

```diff
@@ -87,6 +88,12 @@ public AbstractTask<T> withCondition(@Nonnull Condition condition) {
     return this;
   }
 
+  @Nonnull
+  public AbstractTask<T> withAppend(boolean append) {
```

Comment:
I expect such logic to be immutable. Let's use constructor or builder.

Reply:
Done.